dev-python/internetarchive/internetarchive-5.3.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{11..13} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1
+
+DESCRIPTION="Lib and CLI for archive.org - for search, uploading, downloading, rename..."
+HOMEPAGE="https://github.com/jjjake/internetarchive"
+SRC_URI="https://github.com/jjjake/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="test"
```

Comment:
Now that you're wiring up tests, this needs to be removed so that tests actually run, if the user has `FEATURES=test`.